src/app/screens/restoreFunds/restoreBtc/index.tsx

```diff
@@ -104,25 +104,25 @@ function RestoreBtc() {
       recipientAddress: string;
       nonOrdinalUtxos: Array<UTXO>;
       accountIndex: number;
-      seedPhrase: string;
-      network: NetworkType;
+      userSeedPhrase: string;
```

Comment:
probably don't need to update this, as we have an open PR for seed vault which will be touching this